receiver/elasticsearchreceiver/metadata.yaml

```
@@ -950,3 +966,28 @@ metrics:
       value_type: int
     attributes: [document_state, index_aggregation_type]
     enabled: false
+  elasticsearch.process.cpu.usage:
+    description: CPU usage in percent, or -1 if not known at the time the stats are computed.
+    unit: '%'
+    gauge:
+      value_type: int
+    attributes: [ ]
+    enabled: false
+  elasticsearch.process.cpu.time:
+    description: CPU time (in milliseconds) used by the process on which the Java virtual machine is running, or -1 if not supported.
```

```suggestion
    description: CPU time used by the process on which the Java virtual machine is running.
```
Let's not report the metric when it's -1.